### DISCUSSION TITLE
Enabling Dependabot

### DISCUSSION THREAD

**Discussion by urashidmalik** [evangwt/grc#1] [Ideas]
Hi @evangwt can you kindly enable dependabot, so we that we keep the package up to date. 
I believe you package has huge potential for anyone using GORM. Love to see dependabot creating Pull request for vulnerable package with remediations

**Comment by evangwt** [comment]
> Hi @evangwt can you kindly enable dependabot, so we that we keep the package up to date. I believe you package has huge potential for anyone using GORM. Love to see dependabot creating Pull request for vulnerable package with remediations
 
The dependabot is enabled now ;)
